docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
     This step helps setup your Azure Devops repo with Cobalt Advocated Pattern Templates that matter to you. These are common instructions that are needed for any audience interested in using Cobalt for infrastructure automation.
 
     * Create a new project
-        * Sign-in to Azure DevOps (https://azure.microsoft.com/en-us/services/devops/)
-        * Select New project and create a name. (ex. Cobalt-Contoso)
-
-            ![New Project](https://user-images.githubusercontent.com/10041279/63442791-4f868600-c3f9-11e9-91f3-c959654f5a1c.png)
+        1. Sign-in to Azure DevOps (https://azure.microsoft.com/en-us/services/devops/)
+        2. Create new project under Organization. First create new Organization if it don't exists already. (ex. Cobalt-Contoso)
 
-        * Select Create
+            ![alt text](./images/Org.png "Create Organization")
+            ![alt text](./images/project.png "Create Project")
 
-> The following CLI command(s) can be run as an alternative to using the portal-based instructions:
+    * Create new repository by fetching source code from the master branch of Cobalt's open-source github project
+        1. Select Repos tab within side-navigation menu
+        1. Select 'Import a repository' from the Repos tab sub-menu and click [Import]
+        1. Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
 
-    
-```bash
-az devops configure --defaults organization="$DEFAULT_ORGANIZATION"
-az devops project create --name "$TEMPLATE_DEVOPS_PROJECT_NAME" --source-control git --visibility private
-az devops configure -d project="$TEMPLATE_DEVOPS_PROJECT_NAME"
-```
-
-* Create new repository by fetching source code from the master branch of Cobalt's open-source github project
-    * Select Repos tab within side-navigation menu
-    * Select 'Import a repository' from the Repos tab sub-menu and click [Import]
-    * Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
-
-        ![Clone Button](https://user-images.githubusercontent.com/10041279/63459072-8ec4cf00-c419-11e9-8ef4-ee7db827e49c.png)
+            ![Clone Button](https://user-images.githubusercontent.com/10041279/63459072-8ec4cf00-c419-11e9-8ef4-ee7db827e49c.png)
 
     * Rename your repository
-        * Click your Repo Name (ex. Cobalt-Contoso) at the top of the page.
+        1. Click your Repo Name (ex. Cobalt-Contoso) at the top of the page.
```

Comment:
why Contoso?

Reply:
I"ll consider using something else but it's a fictional company that has widespread use throughout Microsoft documentation. Continuing the tradition. - https://en.wikipedia.org/wiki/Contoso

Reply:
Most Contoso references have been removed and only left where appropriate.